data/transportation/air/index.html

```diff
@@ -1,135 +1,114 @@
-<p><strong>Abstract:</strong><br />
-This data contains GIS mapping data representing airport locations, aircraft landing facilities, and heliport locations throughout the state of Utah. </p>
-<p><strong>Data Content:</strong><br />
-The following data is available for download:</p>
+<div class="caption">
+  <img title="map" src="{{ '/images/Air_Large.png' | prepend: site.baseurl }}" alt="Air and Heliport Locations" />
+  <p class="caption-text">Airport and Heliport Locations</p>
+</div>
+{% include abstract.html download="https://drive.google.com/drive/folders/0ByStJjVZ7c7mRU90UWh6NEdwVHc"
+  name="Airport and Heliport Locations"
+  stewards="Bureau of Transportation Statistics (BTS)"
+  abstract="This data contains GIS mapping data representing airport locations, aircraft landing facilities, and heliport locations throughout the state of Utah."%}
 <div class="grid package">
-    <div class="grid__col grid__col--12-of-12">
-        <h3 id="AirportLocations">Airport Locations</h3>
-    </div>
-    <div class="grid__col grid__col--12-of-12 package-content">
-<img class="productImage-Thumb" src="{{ "/images/AirPtWithLine_90x90.png" | prepend: site.baseurl }}" alt="Changeme" />Transportation.AirportLocations is a statewide point dataset that contains data from the Geographic Names Information System (GNIS). Attribute information include feature name, feature type, elevation, and USGS 24K scale quadrangle name in which the feature resides.
-Transportation.AirportLocations consists of 111 point features that are categorized by feature type and are representative of airport locations found in the general GNIS dataset. For regional display at 1:100,000.</p>
-<p>The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 coordinate precision)<br />
-The AirportLocations dataset is maintained by the Bureau of Transportation Statistics.</p>
-<p>Questions can be directed to the BTS at (202.366.DATA).</p>
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="AirportLocations">
+      <a href="#AirportLocations" class="anchor--link">Airport Locations</a>
+    </h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="productImage-Thumb" src="{{ '/images/AirPtWithLine_90x90.png' | prepend: site.baseurl }}" alt="Changeme" />
+    Transportation.AirportLocations is a statewide point dataset that contains data from the Geographic Names Information System (GNIS). Attribute information include feature name, feature type, elevation, and USGS 24K scale quadrangle name in which the feature resides. Transportation.AirportLocations consists of 111 point features that are categorized by feature type and are representative of airport locations found in the general GNIS dataset. For regional display at 1:100,000.
+    <p>The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 coordinate precision).<br /> The AirportLocations dataset is maintained by the Bureau of Transportation Statistics.</p>
```

Comment:
Please remove the <br /> and the possible the maintenance sentence since we state above who the steward is.